openprescribing/media/js/src/analyse-map.js

```diff
@@ -206,8 +206,8 @@ return x[1];
 
   getBoundsUrl: function(options) {
     var boundsUrl = config.apiHost + '/api/1.0/org_location/?format=json&';
-    if (options.org === 'CCG') {
-      boundsUrl += 'org_type=ccg';
+    if (options.org !== 'practice') {
+      boundsUrl += 'org_type=' + options.org.toLowerCase();
```

Comment:
Very minor: why not switch the branches so you have `if (options.org === 'practice') { ... } else { ... }`?

Reply:
Yes, fair point I was just being lazy and making the minimal change.